### ISSUE TITLE
Some design cleanup

### ISSUE THREAD

**Issue by tessalt** [tessalt/echo-chamber-js#20]
This is a tricky thing to design for, because: 

- can live anywhere!
- it's doing some style inheritance of the parent page: both font-face and link/button colours are inherited from context

So far it looks pretty like-a-developer-made-it, but if anyone has ideas on how it could be improved visually I'd be happy to hear it!

Also... I need to put up a demo page.

**Comment by una** [comment]
Question: why is there a `main.css` AND a `style.css?` They look almost identical

**Comment by tessalt** [comment]
because I'm negligent. I'll clean up the files and make things more reasonable tonight :smile_cat:

**Comment by kennethormandy** [comment]
Yeah, the style inheritance is really appropriate, nice work on that. Maybe in the demo somewhere there could be scoped sections that show how it does that?

Ex. you could have a Bootstrap section and it would automatically look like Bootstrap forms, a custom section that is purple and has web fonts, an unstyled section that just shows the unstyled HTML or something. Really half-hearted sketch with a pen that was way too large:

![2015-07-15 14 17 37](https://cloud.githubusercontent.com/assets/1581276/8710139/6c3ad912-2afc-11e5-9ba4-2aab456dda1f.jpg)

**Comment by polianajardins** [comment]
~~~~

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by polianajardins** [comment]
~~~~

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by polianajardins** [comment]
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by polianajardins** [comment]
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by polianajardins** [comment]
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by polianajardins** [comment]
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
